members/xCA_L_Grayson.yaml

```diff
@@ -12,7 +12,7 @@ contact_form:
           value: $NAME_FIRST
           required: true
           options:
-            blacklist: ".\\"&,"
```

Comment:
was the intention to blacklist the `"` character, or just an error?

Reply:
just an error. I was trying to escape the backlash in that string